atomicapp/nulecule/container.py

```diff
@@ -93,41 +93,67 @@ def pull(self, image, update=False):
 
         cockpit_logger.info('Skipping pulling docker image: %s' % image)
 
-    def extract(self, image, source, dest, update=False):
+    def extract_files(self, image, source, dest):
         """
-        Extracts content from a directory in a Docker image to specified
+        Extracts a directory/file in a Docker image to a specified
         destination.
 
         Args:
             image (str): Docker image name
             source (str): Source directory in Docker image to copy from
             dest (str): Path to destination directory on host
```

Comment:
Might want to update these comments to say file or directory? - or you can do that later if you want in the "index" PR